modules/api/test/LorisApiRecordingsTest.php

```diff
@@ -0,0 +1,1123 @@
+            gettype($recordingsEventsArray['TaskEvents']['0']['EventSample']),
+            'NULL'
+        );
+        $this->assertSame(
+            gettype($recordingsEventsArray['TaskEvents']['0']['EventType']),
+            'NULL'
+        );
+        $this->assertSame(
+            gettype($recordingsEventsArray['TaskEvents']['0']['TrialType']),
+            'string'
+        );
+        $this->assertSame(
+            gettype($recordingsEventsArray['TaskEvents']['0']['ResponseTime']),
+            'NULL'
+        );
+        $this->assertSame(
+            gettype($recordingsEventsArray['TaskEvents']['0']['EventFilePath']),
+            'string'
+        );
+
+        $this->assertArrayHasKey(
+            'Meta',
+            $recordingsEventsArray
+        );
+        $this->assertArrayHasKey(
+            'CandID',
+            $recordingsEventsArray['Meta']
+        );
+        $this->assertArrayHasKey(
+            'Visit',
+            $recordingsEventsArray['Meta']
+        );
+        $this->assertArrayHasKey(
+            'TaskEvents',
+            $recordingsEventsArray
+        );
+        $this->assertArrayHasKey(
+            '0',
+            $recordingsEventsArray['TaskEvents']
+        );
+        $this->assertArrayHasKey(
+            'Onset',
+            $recordingsEventsArray['TaskEvents']['0']
+        );
+        $this->assertArrayHasKey(
+            'Duration',
+            $recordingsEventsArray['TaskEvents']['0']
+        );
+        $this->assertArrayHasKey(
+            'EventCode',
+            $recordingsEventsArray['TaskEvents']['0']
+        );
+        $this->assertArrayHasKey(
+            'EventSample',
+            $recordingsEventsArray['TaskEvents']['0']
+        );
+        $this->assertArrayHasKey(
+            'EventType',
+            $recordingsEventsArray['TaskEvents']['0']
+        );
+        $this->assertArrayHasKey(
+            'TrialType',
+            $recordingsEventsArray['TaskEvents']['0']
+        );
+        $this->assertArrayHasKey(
+            'ResponseTime',
+            $recordingsEventsArray['TaskEvents']['0']
+        );
+        $this->assertArrayHasKey(
+            'EventFilePath',
+            $recordingsEventsArray['TaskEvents']['0']
+        );
+    }
+
+    /**
+     * Tests the HTTP GET request for the
+     * endpoint /candidates/{candid}/{visit}/recordings/{edffile}/events/meta
+     *
+     * @return void
+     */
+    public function testGetCandidatesCandidVisitRecordingsEdfFileEventsMeta(): void
+    {
+        $response = $this->client->request(
+            'GET',
+            "candidates/$this->candidTest/$this->visitTest/recordings/" .
+            "$this->frecordTest/events/meta",
+            [
+                'http_errors' => false,
+                'headers'     => $this->headers
+            ]
+        );
+        if ($response->getStatusCode() === 404) {
+            $this->markTestIncomplete(
+                "Endpoint not found: GET" .
```

Comment:
should fail

Reply:
It exists when the file actually exists. We need to skip to avoid an error, because it is not in the Docker environment. Thus, not skipping causes Travis to fail

Reply:
Replace this function's code by markTestSkipped

Reply:
This still need to be taken care of.